app/components/TransactionModal.tsx

```diff
@@ -0,0 +1,520 @@
+          currency: "USDC",
+          nativeValue: "NGN 369,723.88",
+          time: "4:14 AM",
+          status: "Completed",
+          fees: "NGN 2,392",
+            recipient: "Francesca Tobiloba",
+            bank: "First Bank of Nigeria",
+            account: "0267856481",
+            memo: "From me, Donda North",
+            fundStatus: "Deposited",
+            timeSpent: "12 seconds",
+            day: "13 May 2024"
+
+        },
+        {
+          id: "t5",
+          type: "Swapped",
+          amount: "15.50",
+          currency: "USDT",
+          nativeValue: "ARS 82,506.45",
+          time: "6:30 PM",
+          status: "Completed",
+          fees: "NGN 2,392",
+            recipient: "Francesca Tobiloba",
+            bank: "First Bank of Nigeria",
+            account: "0267856481",
+            memo: "From me, Donda North",
+            fundStatus: "Deposited",
+            timeSpent: "12 seconds",
+            day: "13 May 2024"
+
+        },
+        {
+          id: "t6",
+          type: "Swapped",
+          amount: "200.00",
+          currency: "USDT",
+          nativeValue: "KES 7,890.65",
+          time: "8:00 AM",
+          status: "Completed",
+          fees: "NGN 2,392",
+            recipient: "Francesca Tobiloba",
+            bank: "First Bank of Nigeria",
+            account: "0267856481",
+            memo: "From me, Donda North",
+            fundStatus: "Deposited",
+            timeSpent: "12 seconds",
+            day: "13 May 2024"
+
+        },
+        {
+          id: "t7",
+          type: "Swapped",
+          amount: "75.30",
+          currency: "USDC",
+          nativeValue: "NGN 53,674.20",
+          time: "5:55 PM",
+          status: "Failed",
+          fees: "NGN 2,392",
+            recipient: "Francesca Tobiloba",
+            bank: "First Bank of Nigeria",
+            account: "0267856481",
+            memo: "From me, Donda North",
+            fundStatus: "Deposited",
+            timeSpent: "12 seconds",
+            day: "13 May 2024"
+
+        },
+      ],
+    },
+    {
+      id: "3",
+      date: "2 days ago",
+      items: [
+        {
+          id: "t8",
+          type: "Swapped",
+          amount: "28.10",
+          currency: "USDC",
+          nativeValue: "NGN 369,723.88",
+          time: "4:14 AM",
+          status: "Completed",
+          fees: "NGN 2,392",
+            recipient: "Francesca Tobiloba",
+            bank: "First Bank of Nigeria",
+            account: "0267856481",
+            memo: "From me, Donda North",
+            fundStatus: "Deposited",
+            timeSpent: "12 seconds",
+            day: "13 May 2024"
+
+        },
+      ],
+    },
+  ];
+
+  const slideUpAnimation = {
+    initial: { y: "100%", opacity: 0 },
+    animate: { y: 0, opacity: 1 },
+    exit: { y: "100%", opacity: 0 },
+    transition: {
+      type: "spring",
+      stiffness: 300,
+      damping: 30,
+      duration: 0.2,
+    },
+  };
```

Comment:
you can move this to `app\components\AnimatedComponents.tsx` and import for usage here.